source/css/_settings.themes.scss

```diff
@@ -88,21 +88,20 @@ $themes: (u-theme--treefrog: (darker: #1b2d00, dark: #355724, base: $c-theme-tre
       &::after {
         content: "";
         display: block;
+        position: absolute;
         height: 100%;
         width: 100%;
+        width: calc(100vw / 7 * 1 - 14px);
```

Comment:
@davideleuterius This `- 14px` is the problem. If that gets removed, it works fine. 

What is the reason for having that in there?

Reply:
@designerbrent I have the exact opposite problem. I need that 14px in there to have it flush, without the gap. I am not sure why this is different for us. I'll remove that and if it looks good on your end I think that might be good then?